### ISSUE TITLE
Allegro Archive parsing

### ISSUE THREAD

**Issue by TetSigma** [allegro/allegro-api#7241]
I need to parse 600 offer id's on allegro archive. Default API doesnt allow it as far as i know. In allegro terms and conditions i only found out that sellers cant parse other sellers info. But nothing about parsing archive (all the offer id's we have belong to our account on allegro). Can i do it?

**Comment by Lukasz-Zurek** [comment]
Unfortunately we don't provide that kind of functionality in our API. Archived offers are only available on https://archiwum.allegro.pl

**Comment by TetSigma** [comment]
Yes i know that you dont provide such kind of functionality, will i get in trouble if i parse archiwum.allegro myself with beautifullsoup?

(also we can try to speak in polish if you dont understand what im saying)

**Comment by TetSigma** [comment]
> Unfortunately we don't provide that kind of functionality in our API. Archived offers are only available on https://archiwum.allegro.pl

Yes i know that you dont provide such kind of functionality, will i get in trouble if i parse archiwum.allegro myself with beautifullsoup?

(also we can try to speak in polish if you dont understand what im saying)

**Comment by Lukasz-Zurek** [comment]
I didn't understand that you did mean https://archiwum.allegro.pl/ service, as this forum is about Allegro API so it is out of our hands. Despite the fact that this question is not API related - all I can say is the fact that you cannot parse any data. Sooner or later you are going to be automatically blocked.

**Comment by TetSigma** [comment]
Ok thanks
